apps/posterizarr/entrypoint.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+set -e
+
+# Check for test mode
+if [ "$1" = "--test" ] || [ "$POSTERIZARR_TEST_MODE" = "true" ]; then
+    echo "Running in test mode - skipping PowerShell script execution"
+    echo "Container is ready for testing"
+
+    # Keep container running for tests
+    if [ "$1" = "--test" ]; then
+        echo "Container will stay alive for testing"
+        # Sleep indefinitely to keep container running for tests
+        exec tail -f /dev/null
+    fi
+    exit 0
+fi
```

Comment:
I don't see why this is needed, if you want to override the entrypoint you can do so using the CRI options e.g.

```docker run --entrypoint="" docker.io/library/app:latest tail -f /dev/null```